src/lib.rs

```diff
@@ -533,7 +533,7 @@ pub use offset::LocalResult;
 #[doc(inline)]
 pub use offset::{FixedOffset, Offset, TimeZone, Utc};
 
-mod round;
+pub mod round;
```

Comment:
I don't understand the argument put forth in the commit message. What's the rationale for this change?

Reply:
The idea is to highlight the importants traits in the "Traits" section: `TimeZone, `Offset`, `Datelike` and `Timelike`.
I don't think `DurationRound` and `SubsecRound` are good enough to be part of the list in the main documentation.
We can only hide it in the list if the `round` module is public.

To say it stronger, in my opinion the `DurationRound` trait is something that begs for a replacement.

Reply:
Okay, sure, but if that's the goal IMO the other changes to that goal should be in the same commit as that. In general, the first commit here seems too large/unfocused (it probably doesn't help that I'm completely unfamiliar with the semantics of `doc(no_inline)`)...

Reply:
O, sorry. That was only the goal of that commit. No: the goal of the PR was tweaking `doc(incline)` and `doc(no_inline)`.

If the original module is public:
- a re-export comes under the list "Re-exports".
- `doc(incline)` on a re-export copies the documentation, so it now lives at two places.

If the original module is private:
- a re-export moves the documentation.
- `doc(no_incline)` makes the documentation unreachable.